packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -21,6 +23,17 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
     ], this.db.collection(this.collectionName)).catch((e: any) => {
       throw e;
     });
+    if (customConflictEngine !== undefined) {
+      this.conflictEngine = customConflictEngine;
+    } else {
+      this.conflictEngine = new TimestampConflictEngine();
```

Comment:
Discussion - conflicts enabled by default.
I think we should simply assign value and do not setup default conflict engine. Conflict engine will affect queries etc. so not anyone will want to have it when not needing this feature.

Reply:
Agreed :+1: